.github/workflows/commit-lint.yml

```diff
@@ -15,4 +15,10 @@ jobs:
         with:
           python-version: '3.x'
       - run: pip install gitlint
-      - run: gitlint --commits $(git merge-base origin/master HEAD)..HEAD
+      - run: |
+          gitlint \
+            -c general.ignore-fixup-commits=false \
+            -c general.ignore-fixup-amend-commits=false \
+            -c general.ignore-squash-commits=false \
+            --contrib contrib-body-requires-signed-off-by,contrib-disallow-cleanup-commits \
+            --commits "$(git merge-base origin/master HEAD)..HEAD"
```

Comment:
I'd rather put a `.gitlint` file in the repository than using CLI flags, so contributors can run gitlint locally!

Reply:
Bigger than expected with one commit? Hahah you clearly haven't seen my 800+ commit PRs yet.

But sure, go ahead! I'm not the maintainer here, so I have nothing to say anyways! :laughing: